desktop/components/split_test/running_tests.coffee

```diff
@@ -14,4 +14,9 @@
 # this should export empty Object
 # module.exports = {}
 
-module.exports = {}
+module.exports =
```

Comment:
Actually, I'm thinking a better progression might be to have the last PR for this feature be the thing that actually unveils the feature and adds the AB test.

In the meantime, you may want to iterate over several PR's, and get QA from other people. So maybe to start we can just have it triggered by a query param (+ that way we can test on a few different pages), and then once we're happy, the next PR could be the analytics/AB test? 

That should then be simple, just replacing all the checks against the query param with the `sd` var from the test. (+ 'experiment viewed' analytics events).